paddle/understand_sentiment_lstm.py

```diff
@@ -147,13 +147,15 @@ def run_benchmark(model, args):
     place = fluid.CPUPlace() if args.device == 'CPU' else fluid.GPUPlace(0)
     exe = fluid.Executor(place)
     exe.run(fluid.default_startup_program())
-    for it, pass_id in enumerate(xrange(args.pass_num)):
+    
+    iterator = 0
+    for pass_id in xrange(args.pass_num):
         accuracy.reset(exe)
-        if iter == args.iterations:
-            break
         for data in train_reader():
             chopped_data = chop_data(
                 data, chop_len=args.seq_len, batch_size=args.batch_size)
+            if len(data) < args.batch_size : continue
```

Comment:
>   File "/paddle/Paddle_release/Paddle/python/paddle/v2/fluid/executor.py", line 144, in run
>    self.executor.run(program.desc, scope, 0, True)
> EnforceNotMet: enforce capacity == in_size failed, 81920 != 20480

Maybe this is a bug of operator.  I do not think it needs to skip here.

Reply:
I have notified the relevant person. https://github.com/PaddlePaddle/Paddle/issues/6533